### PR TITLE
Add Open Science Resources

### PR DESCRIPTION
**Pull request by asoplata**
I would like to add my [Open Science Resources](https://github.com/asoplata/open-science-resources) list to the Awesome list, since there appears to be no specific precursor for it. It is intended to be cross-disciplinary, and has many links to open science data and many other resources. Thanks for any consideration!

**https://github.com/asoplata/open-science-resources**

**There is no cross-disciplinary list for doing open science in general already present. Much of the barrier to working with open science data or using open science tools is simple ignorance of the depth and breadth of such resources. By compiling these resources together, including other meta-resources, I've tried to provide a good starting place for any scientists interested in hunting for open data in their field, or anyone who is interested in contributing to a field.**

Note: I'd like to mention that the below contribution instructions aren't the same as what's on the main [Contribution Guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) page? The below list has some extra recommendations that would be useful to add to that.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by asoplata** [comment]
I believe I have updated both the PR patches and the list itself to fully comply with both the guidelines and the "pull request template". If there are still outstanding issues, please let me know what they are. Thank you for your time in considering this!

File: readme.md [hunk 1]
```diff
@@ -588,7 +588,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit) - Open source Java app performance management tool.
-- [Open Science Resources](https://github.com/asoplata/open-science-resources) - Open scientific data repositories, meta-lists of data repositories, tools, and much more for doing science more openly.
+- [Open Science Resources](https://github.com/asoplata/open-science-resources) - Anyone can learn and contribute to science using publicly-accessible data and tools, no PhD required!
```
Comment (legodude17) [inline readme.md]: Explain what "Open Science" means.

**Comment by asoplata** [comment]
The description of open science should now be clear enough.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by beardicus** [comment]
Hi! I've reviewed this submission and it has a few issues.

✅: Good!
❌: Needs work!
:grey_question:: Not sure, not required, or mixed results

_**Reviewer comments are in bold italics after an item**_

# Awesome Requirements

- ✅Has been around for at least 30 days.
- ✅ It's the result of hard work and the best I could possibly produce. _**Looks like a good selective list of resources**_
- ✅ Non-generated Markdown file in a GitHub repo.
- ✅ Includes a succinct description of the project/theme at the top of the readme. _**Description is a bit verbose**_
- ✅ The repo should have `awesome-list` & `awesome` as GitHub topics.
- ✅ Not a duplicate.
- ✅ Only has awesome items. Awesome lists are curations of the best, not everything.
- :grey_question: Includes a project logo/illustration whenever possible. _**Not required**_
- ❌ Entries have a description, unless the title is descriptive enough by itself. _**Mixed. Many have no description and could definitely use one**_
- ✅ Includes the [Awesome badge]
	- ✅ Should be placed on the right side of the readme heading.
	- ✅ Should link back to this list.
- ✅ Has a Table of Contents section.
	- ✅ Should be named `Contents`, not `Table of Contents`.
	- ❌ Should be the first section in the list.  _**It is first but is not formatted like a section. "Contents:" should be a header like all the other sections, not a first level bullet item**_
	- ✅ Should only have one level of sub-lists, preferably none.
- ✅ Has an appropriate license.
- ✅ Has contribution guidelines.
	- ❌ The file should be named `contributing.md`. Casing is up to you. _**No markdown extension**_
- ❌ Has consistent formatting and proper spelling/grammar. _**Internally consistent but not consistent with the standard. Item descriptions should not be a link**_
	- ✅ The link and description are separated by a dash.
	- ❌ The description starts with an uppercase character and ends with a period. _**No periods used**_
	- ✅ Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- ✅ Doesn't include a Travis badge.

# Additional Comments

- This repo should be named `awesome-open-science` in my opinion.
- Section headers should be h2 (`## Header`) not h3.

Thanks for putting together this list it looks nice but needs some tweaks! Feel free to @ me if you update and I can re-review.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/asoplata/open-science-resources
✖ Linting

  README.md:27:5
  ✖    1:1    Missing file contributing.md                                                         remark-lint:awesome/contributing
  ✖    1:1    Missing License section                                                              remark-lint:awesome/license
  ✖    1:1    Missing or invalid Table of Contents                                                 remark-lint:awesome/toc
  ✖   24:1    Rules should use ---                                                                 remark-lint:rule-style
  ✖   27:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   27:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   28:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   28:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   29:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   29:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   30:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   30:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   31:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   31:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   32:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   32:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   33:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   33:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   34:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   34:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   35:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   35:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   36:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   36:7    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖   38:1    Rules should use ---                                                                 remark-lint:rule-style
  ✖   46:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   51:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖   81:3    Link to http://scisoft-net-map.isri.cmu.edu/ is dead                                 remark-lint:no-dead-urls
  ✖  116:3    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖  118:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  120:3    Link to http://cbiit.nci.nih.gov/ncip/biomedical-informatics-resources/data is dead  remark-lint:no-dead-urls
  ✖  126:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  152:3    Invalid list item link URL                                                           remark-lint:awesome/list-item
  ✖  154:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  191:3    Link to https://idash.ucsd.edu/ is dead                                              remark-lint:no-dead-urls
  ✖  205:117  List item link and description must be separated with a dash                         remark-lint:awesome/list-item
  ✖  231:3    Link to http://phenoexplorer.org/ is dead                                            remark-lint:no-dead-urls
  ✖  238:45   List item link and description must be separated with a dash                         remark-lint:awesome/list-item
  ✖  239:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  240:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  241:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  242:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  243:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  244:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  245:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  246:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  254:3    Link to https://library.tmc.edu/website/whole-brain-catalog/ is dead                 remark-lint:no-dead-urls
  ✖  262:93   List item link and description must be separated with a dash                         remark-lint:awesome/list-item
  ✖  283:3    Link to https://www.helmholtz.de/en/research/open_access/ is dead                    remark-lint:no-dead-urls
  ✖  285:3    Link to https://www.opendatacon.org/ is dead                                         remark-lint:no-dead-urls
  ✖  289:5    Incorrect indentation before bullet: remove 2 spaces                                 remark-lint:list-item-bullet-indent
  ✖  325:50   List item link and description must be separated with a dash                         remark-lint:awesome/list-item
  ✖  326:38   List item link and description must be separated with a dash                         remark-lint:awesome/list-item

  53 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**